src/main/java/com/corndel/cashregister/Level2.java

```diff
@@ -1,28 +1,63 @@
 package com.corndel.cashregister;
 
 import com.corndel.cashregister.models.Item;
+
+import java.util.ArrayList;
 import java.util.List;
 
 public class Level2 {
-  /**
-   * Goes through the given drawer and counts how many *coins* there are in total
-   * 
-   * <p>
-   * N.b. just the number of coins, not the total value of them
-   * 
-   * <p>
-   * N.b. $1 is a note, not a coin
-   */
-  public static int countCoins(List<Item> drawer) {
-    // TODO
-    return 0;
-  }
+    /**
+     * Goes through the given drawer and counts how many *coins* there are in total
+     *
+     * <p>
+     * N.b. just the number of coins, not the total value of them
+     *
+     * <p>
+     * N.b. $1 is a note, not a coin
+     */
+    public static int countCoins(List<Item> drawer) {
+        // TODO
+        int coinCount = 0;
+
+//    var coins = new ArrayList<>();
```

Comment:
I like your thinking here you could add the items to a list and check the items in drawer against that. 

```java
    public static int countCoins(List<Item> drawer) {
        // TODO
        int coinCount = 0;

        List<String> coins = new ArrayList<>();
        coins.add("penny");
        coins.add("nickel");
        coins.add("dime");
        coins.add("quarter");


        for (var item : drawer) {
            if (coins.contains(item.name)) {
                coinCount += item.quantity;
            }
        }
        return coinCount;
    }
```